extensions/amp-list/amp-list.md

```diff
@@ -220,6 +220,23 @@ In several cases, we may need the `<amp-list>` to resize on user interaction. Fo
 </amp-list>
 ```
 
+### Initialization from amp-state
+
+In some cases, it may be desirable to have your `<amp-list>` component initialize off of `<amp-state>` rather than from a json endpoint.
+You may do that by utilizing the `amp-state:` protocol in the `src` attribute. The data in state must follow the same rules as the json that
+would have been retrieved from an endpoint. For example,
+
+```html
+<amp-state id="listExample">
+  <script type="application/json">
+    {
+      "items": [{"id": 1}, {"id": 2}, {"id": 2}]
+    }
+  </script>
+</amp-state>
+<amp-list src="amp-state:listExample">...</amp-list>
```

Comment:
Let's make this example as simple as possible but also complete enough to be copy/pasted.

Nit: `s/listExample/localState`?

Reply:
Done

Reply:
Going to double check that this works actually (via copy/paste)